physicsnemo/distributed/shard_utils/conv_patches.py

```diff
@@ -490,12 +488,6 @@ def backward(
             **conv_kwargs,
         )
 
-        # Synchronize weight and bias gradients across all ranks
-        group = spec.mesh.get_group()
-        dist.all_reduce(grad_weight, group=group)
-        if grad_bias is not None:
-            dist.all_reduce(grad_bias, group=group)
-
```

Comment:
This is handled automatically by DTensor, actually, and no longer needed here.  It wasn't wrong, though when the mesh is 2D it breaks.